### ISSUE TITLE
Condition: Items not working

### ISSUE THREAD

**Issue by duncanwebb** [TeamPneumatic/pnc-repressurized#702]
### Minecraft Version
1.16.4
### Forge Version
35.1.29
### Mod Version
pneumaticcraft-repressurized-1.16.4-2.8.3-109

### Describe your problem, including steps to reproduce it
The condition piece items returns false when the condition should be true.

This is an extract of Tek Stack's green house building program, and it is checking for 1024 item of cobblestone in a chest, this is actually true.

https://user-images.githubusercontent.com/4198031/103481033-7409b500-4dd8-11eb-851d-60b39fbfac45.mp4

![2021-01-03_15 20 47](https://user-images.githubusercontent.com/4198031/103481103-df538700-4dd8-11eb-8332-d3bd0ebe2be3.png)

![image](https://user-images.githubusercontent.com/4198031/103481068-ac10f800-4dd8-11eb-8491-091af537bd47.png)

### Any other comments?
The drone has 35 inventory upgrades, so it should be able to import both tools. But it is only importing the Netherite pickaxe.

You can't easily see the numbers on the sign with the GitHub player, but you can with an external player. The measure variable says 0, 0, 0.

**Comment by desht** [comment]
Works fine for me.  I set up a smart chest full of advanced pressure tubes, and ran this program on it: https://pastebin.com/T5Xi26An

It correctly checks for 4000 or more tubes and writes the actual number found on the sign (4608 for a full smart chest: 64 * 72)

Also tested removing some tubes, adding some other items... all fine.

**Comment by duncanwebb** [comment]
Indeed, your program works just fine. And the condition is the same an in the greenhouse program.

I wonder if this is related to the original program was written for 1.7.10. I'll delete and add the components back in again.

Many thanks for testing this.

**Comment by desht** [comment]
I imported Tekstack's program (https://pastebin.com/qLMM2w04 right?) and one thing I did notice is that "Match Durability" is enabled on the cobblestone's (and other blocks) item filter, whereas if you create a new item filter with cobblestone, it will (correctly) be in "Ignore Durability" mode.  And of course the checkbox is greyed out, because cobblestone items don't have durability.

But of course back in 1.12 and earlier, items had the metadata byte, which was effectively also durability.  So yeah I'm pretty sure this is an artifact of importing programs from an earlier release.  Since I do want to preserve backwards compat as much as possible, I think I can see a fix for this...
